src/ember-app.js

```diff
@@ -32,7 +32,7 @@ class EmberApp {
     let distPath = path.resolve(options.distPath);
     let config = this.readPackageJSON(distPath);
 
-    this.appFilePath = config.appFile;
+    this.appFilesPath = config.appFiles;
```

Comment:
Small nit-pick, I'd prefer `this.appFilePaths` vs `this.appFilesPath`, what do you think?